src/util/Structures.js

```diff
@@ -20,7 +20,6 @@
  * * **`Role`**
  * * **`User`**
  * * **`CommandInteraction`**
- * * **`MessageComponentInteraction`**
```

Comment:
You also need to remove this line then:
https://github.com/discordjs/discord.js/blob/a56fa6dd7b5386bc4cbc44bc4e88f109e3419bb5/src/util/Structures.js#L115

Reply:
It has been removed. Thanks for the review.